models/appModels.js

```diff
@@ -76,6 +76,33 @@ const fetchSpecificCommentsByArticleId = (id) => {
     })
 }
 
-module.exports = {fetchTopics, fetchArticles, fetchSpecificArticle, fetchSpecificComments, fetchSpecificCommentsByArticleId}
+const updateVotes = (article_id, inc_votes) => {
+
+    const queryString = `SELECT votes FROM articles WHERE article_id = $1;`
+    let newVotes;
+
+    return db.query(queryString, [article_id]).then(({rows}) => {
```

Comment:
Instead of having this JS logic in here, in your SQL query you can SET votes = votes + $1 and that way you don't need to do a db query first to get the current value of the votes